clients/admin-ui/src/features/poc/privacy-notices-sandbox/components/ExperienceConfigSection.tsx

```diff
@@ -0,0 +1,116 @@
+import {
+  AntButton as Button,
+  AntFlex as Flex,
+  AntInput as Input,
+  AntTypography as Typography,
+} from "fidesui";
+import palette from "fidesui/src/palette/palette.module.scss";
+
+import type { PrivacyNoticeResponse } from "~/types/api";
+
+import PreviewCard from "./PreviewCard";
+
+interface ExperienceConfigSectionProps {
+  region: string;
+  onRegionChange: (region: string) => void;
+  onFetchExperience: () => void;
+  isLoading: boolean;
+  errorMessage: string;
+  privacyNotices: PrivacyNoticeResponse[];
+}
+
+/**
+ * Renders a list of privacy notices with their children
+ */
+const PrivacyNoticesList = ({
+  privacyNotices,
+}: {
+  privacyNotices: PrivacyNoticeResponse[];
+}) => {
+  if (privacyNotices.length === 0) {
+    return null;
+  }
+
+  return (
+    <div className="max-h-60 overflow-y-auto">
+      <ul className="m-0 list-inside list-disc pl-2">
+        {privacyNotices.map((notice) => (
+          <li key={notice.id} className="list-none">
+            <ul className="m-0 list-inside list-disc">
+              <li className="py-1">{notice.name}</li>
+              {notice.children &&
+                notice.children.length > 0 &&
+                notice.children.map((child) => (
+                  <li key={child.id} className="py-1 pl-5">
+                    {child.name}
+                  </li>
+                ))}
+            </ul>
+          </li>
+        ))}
+      </ul>
+    </div>
+  );
+};
+
+const ExperienceConfigSection = ({
+  region,
+  onRegionChange,
+  onFetchExperience,
+  isLoading,
+  errorMessage,
+  privacyNotices,
+}: ExperienceConfigSectionProps) => {
+  return (
+    <Flex vertical gap="small">
+      <Typography.Title level={3}>Configuration</Typography.Title>
+
+      <Flex gap={24}>
+        {/* Left Column - Configuration */}
+        <Flex vertical flex={1} style={{ minWidth: 0 }}>
+          <Typography.Text strong className="mb-2 block">
+            Experience region
+          </Typography.Text>
+          <Flex gap={8} className="mb-4">
```

Comment:
8 is currently the same as "small." Prefer set sizes over arbitrary when possible.
```suggestion
          <Flex gap="small" className="mb-4">
```